CMSgov/cmcs-eregulations
# EREGCSC-1174 Prototype Data Access Layer

Resolves [EREGCSC-1174 [Prototype] Create Data Access Layer](https://jiraent.cms.gov/browse/EREGCSC-1174)

**Description**

Create API layer containing methods to fetch data from server and transform to desired shape for use in Vue components.

**This pull request changes:**

- Adds an api.js file with methods to pull down part names, most recent date of update, and homepage ToC information
- Displays most recent date of update in footer
- Homepage table of contents is rendered and Part Link takes the user to a part page via Vue Dynamic Route Matching
- Enables JumpTo Vue component to navigate to part page via Vue Dynamic Route Matching

**Steps to manually verify this change:**

1. Visit [deployed prototype](https://d2hgxe2x7z11zt.cloudfront.net/) and ensure all above items are present.
